java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -0,0 +1,164 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.ZaddOptions;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Sorted set Commands interface.
+ *
+ * @see <a href="https://redis.io/commands/?group=sorted-set">Sorted set Commands</a>
+ */
+public interface SortedSetBaseCommands {
+    /**
+     * Adds members with their scores to the sorted set stored at <code>key</code>. If a member is
+     * already a part of the sorted set, its score is updated.
+     *
+     * @see <a href="https://redis.io/commands/zadd/">redis.io</a> for more details.
+     * @param key - The key of the sorted set.
+     * @param membersScoresMap - A mapping of members to their corresponding scores.
+     * @param options - The Zadd options.
+     * @param changed - Modify the return value from the number of new elements added, to the total
+     *     number of elements changed.
+     * @returns The number of elements added to the sorted set. If <code>changed</code> is set,
+     *     returns the number of elements updated in the sorted set. If <code>key</code> holds a value
+     *     that is not a sorted set, an error is returned.
+     * @example
+     *     <p><code>
+     * client.zadd("mySortedSet", Map.of("member1", 10.5d, "member2", 8.2d), ZaddOptions.builder().build(), false).get();
+     * 2 (Indicates that two elements have been added or updated in the sorted set "mySortedSet".)
+     *
+     * client.zadd("existingSortedSet", Map.of("member1", 15.0d, "member2", 5.5d), ZaddOptions.builder().conditionalChange(ZaddOptions.ConditionalChange.ONLY_IF_EXISTS).build(), false).get();
+     * 2 (Updates the scores of two existing members in the sorted set "existingSortedSet".)
```

Comment:
```suggestion
     * client.zadd("mySortedSet", Map.of("member1", 10.5d, "member2", 8.2d), ZaddOptions.builder().build(), false).get();
     * // 2 (Indicates that two elements have been added or updated in the sorted set "mySortedSet".)
     *
     * client.zadd("existingSortedSet", Map.of("member1", 15.0d, "member2", 5.5d), ZaddOptions.builder().conditionalChange(ZaddOptions.ConditionalChange.ONLY_IF_EXISTS).build(), false).get();
     * // 2 (Updates the scores of two existing members in the sorted set "existingSortedSet".)
```

I'm lazy to find a ref, but we were asked to keep examples as a compilable code